torch/utils/data/datapipes/iter/combinatorics.py

```diff
@@ -67,6 +67,14 @@ class ShufflerIterDataPipe(IterDataPipe[T_co]):
         buffer_size: The buffer size for shuffling (default to ``10000``)
         unbatch_level: Specifies if it is necessary to unbatch source data before
             applying the shuffle
+
+    Example:
+        >>> import random
+        >>> from torchdata.datapipes.iter import IterableWrapper
+        >>> dp = IterableWrapper(range(10))
+        >>> random.seed(123)
```

Comment:
Let's remove `seed` part for now. (I am not sure if we rely allow such seeding method in the future)